api/src/main/java/org/openmrs/module/fhirExtension/dao/impl/TaskDaoImpl.java

```diff
@@ -23,11 +23,14 @@
 @Repository
 public class TaskDaoImpl implements TaskDao {
```

Comment:
any specific reason of using this instead of https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirTaskDaoImpl.java?

Reply:
Im not very sure of the reasoning, the only thing I can see is that the extension class supports additional methods